engine/apps/slack/scenarios/paging.py

```diff
@@ -753,7 +753,25 @@ def _get_selected_entries_list(input_id_prefix, key, entries):
 
 def _display_availability_warnings(payload, warnings, organization, user):
     metadata = json.loads(payload["view"]["private_metadata"])
+    return _get_availability_warnings_view(
```

Comment:
This is moved from [here](https://github.com/grafana/oncall/pull/2562/files#diff-ba09b07d53839aef9f2a7f3b6cf2a4a8c50e62e53f3779ed92c401961f941a81L788-L797), so there's a new function `_get_availability_warnings_view` that can be reused.